core/src/main/scala/chisel3/Data.scala

```diff
@@ -190,6 +191,9 @@ object ActualDirection {
       None
     }
   }
+
+  implicit def lookupableDirection: Lookupable.Simple[ActualDirection] =
+    Lookupable.isLookupable[ActualDirection]
```

Comment:
Please add this to `object Lookupable` along with the other ones, also good to add a sanity test, see https://github.com/chipsalliance/chisel/pull/4497 for an example. It would be reasonable to merge a lot of the "simple lookupable" tests into a single test instead of separate ones for each.